_posts/2020-05-06-welcome-to-webarkit.markdown

```diff
@@ -20,7 +20,7 @@ Worth noting is the great work that Thorsten Bux and Daniel Fernandes have done
 
 Unfortunately as regards **WebARStudio**, some improvements have been made but not yet for the NFT markers, it is in fact necessary to develop the part of the code to support the version with Web Worker required for Mobile devices. I hope to accomplish this upgrade in a timely, because it is a long awaited feature…
 
-An interesting mention should be made for the code developed by Itamayo for the rotation of the camera and a renderer for graphic objects developed by Daniel Fernandes based on CSS3D rendering by Threejs, many interesting possibilities open up with these two projects !!
```

Comment:
@itamayo i decided to delete for now that mention, maybe we can tell about this, another time if and when it will have some further development.